cabra/src/pages/components/TasksCalendar.tsx

```diff
@@ -0,0 +1,199 @@
+import "@fullcalendar/react/dist/vdom";
+
+import FullCalendar, { EventContentArg, EventInput } from "@fullcalendar/react";
+import { ITask, TaskStatus } from "../../types/task";
+import tw, { styled } from "twin.macro";
+
+import { Link } from "react-router-dom";
+import { PlusIcon } from "@heroicons/react/outline";
+import { add } from "date-fns";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import { routeHelpers } from "../../routes";
+import { useMemo } from "react";
+
+interface Props {
+  tasks: ITask[] | null;
+  isLoading: boolean;
+}
+
+const Container = styled.div(tw`p-4 rounded-lg text-stone-50 bg-primary`);
+const CalendarWrapper = styled.div`
+  .fc-toolbar-chunk {
+    ${tw`flex items-center gap-x-4`}
+  }
+
+  .fc-toolbar-title {
+    ${tw`!m-0 text-xl font-bold`}
+  }
+
+  .fc-next-button,
+  .fc-prev-button,
+  .fc-today-button {
+    ${tw`!m-0 px-2 !bg-secondary !shadow-lg !outline-none !border-none cursor-pointer`}
+    ${tw`transform transition-transform hover:scale-105 disabled:(scale-100 cursor-default)`}
+  }
+
+  .fc-today-button {
+    ${tw`capitalize font-bold`}
+  }
+
+  .fc-day {
+    ${tw`text-sm text-left font-normal`}
+  }
+  .fc-theme-standard {
+    td,
+    th,
+    .fc-scrollgrid {
+      ${tw`border-none`}
+    }
+  }
+
+  .fc-scrollgrid-sync-table {
+    ${tw`border-separate`}
+
+    tbody {
+      ${tw`space-y-1`}
+    }
+
+    tr {
+      ${tw`space-x-1`}
+    }
+  }
+
+  .fc-daygrid-day {
+    ${tw`p-0.5 relative`}
+
+    .add-task-link {
+      ${tw`hidden`}
+    }
+
+    &:hover {
+      .add-task-link {
+        ${tw`grid`}
+      }
+    }
+
+    .fc-daygrid-day-frame {
+      ${tw`rounded bg-secondary`}
+    }
+
+    .fc-daygrid-day-top {
+      ${tw`flex-row`}
+    }
+
+    &.fc-day-other {
+      .fc-daygrid-day-frame {
+        ${tw`bg-tertiary`}
+      }
+
+      .fc-daygrid-day-top {
+        ${tw`opacity-100 text-black`}
+      }
+    }
+
+    &.fc-day-today {
+      ${tw`bg-cyan-400 rounded`}
+    }
+  }
```

Comment:
I think this color is too bright. I suggest try `lightBlue-500`.
```suggestion
    &.fc-day-today {
      ${tw`bg-lightBlue-500 rounded`}
    }
  }
```